Contributing/How-to-Assist-with-Fundraising.md

```diff
@@ -1,8 +1,9 @@
-## Raise Funds:  
+## Raise Funds:
 
-1. moja global is not for profit and runs on voluntary contributions (cash and in-kind). You can find more information about [who pays here](https://docs.google.com/document/d/15KY1CkCxRzB9ZBp68OLc2eWYuC5W-rqCKnTLaIG4tbg/edit?usp=sharing).
-1. If you would like to make a financial contribution or want to help raise funds, please send an email to info@moja.global 
-1. Use a clear and descriptive title
-1. Please provide background or references to prove your good intentions if you do not have a track record on GitHub
-1. Please allow a few days for *moja global* to reply
- 
+moja global supports new contributors and people willing to help simplify our processes and workflow. If you would like to help us with communication and outreach tasks, please have a look at the following points to get started:
+
+1.  moja global is a not-for-profit organization and runs on voluntary contributions through cash and in-kind. You can find more information about our funding pipeline through this [document](https://docs.google.com/document/d/15KY1CkCxRzB9ZBp68OLc2eWYuC5W-rqCKnTLaIG4tbg/edit?usp=sharing).
+2.  If you would like to make a financial contribution or want to help raise funds, please send us an email at [info@moja.global](mailto:info@moja.global).
+3.  Use a clear and descriptive title.
+4.  Please provide background or references to prove your good intentions if you do not have a track record on GitHub.
```

Comment:
Fundraising and GitHub don't go well together I suppose.
@aornugent some of the Governance documents may help us here. 😅

Reply:
What do you mean Shubham? Risk of someone changing the approved process? Where else can we make the rules and approvals in the linked doc available for cross reference?

Reply:
I meant that the discussions regarding Fundraising would not be on GitHub right? Then it does not make sense to ask for a track record on GitHub on Point 4 to help with Fundraising. 
I do not think that there is any problem with the doc itself. 😅

Reply:
Oh got it! You're right we can probably remove point 4

Reply:
It'd be cool to link to this somehow, see section 8. Shared cost of open source. I've asked Kyle about hosting it on the moja mainpage. @HarshCasper can you please document it and the moja global Charter in the content strategy? 

https://docs.google.com/document/d/1Q2aVTzWchXQwupbEN7s37xANkPZvrLdfaFyYOT_8Diw/edit?usp=drivesdk

Reply:
Done!